site/resources.md

```diff
@@ -33,20 +33,17 @@ Resources
   _add description_
 
 ### Data Analysis
-* [Numpy (basics)](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/numpy.ipynb)
-
+* [**Numpy (basics)**](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/numpy.ipynb)
   This notebook highlights pythonic programing with numpy arrays as opposed to lists.
 
-* [Pandas and Numpy (csv data)](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/dataAccess/numpy.ipynb)
-
+* [**Pandas and Numpy (csv data)**](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/dataAccess/numpy.ipynb)
```

Comment:
Just about, these explicitly need two spaces at the end of the link -- it's a markdown formatting thing, otherwise it doesn't render correctly for the webpage.
Two spaces after the link meaning: "...ipynb)  "

Reply:
Holy moly, I see now! I didn't know that markdown inserted a `<br>` when a line ends with two spaces. I've always put a blank line between a bullet item and description (like the way I did the initial ones with the `__add description__` text), so I never ran into that. I guess this is how we achieve the tight spacing between the bullet point and the description. Thank you @srcarter3! Should be good to go now.